booking.js

```diff
@@ -178,8 +202,15 @@ const createParticipants = function (minPar, maxPar) {
         const participantOption = document.createElement("option");
         participantOption.classList.add("partOption");
         participantOption.value = i;
-        participantOption.innerText = i;
-        particiSelect.appendChild(participantOption);
+        if(i == 1){
+            participantOption.innerText = `${i} Participant`;
+            particiSelect.appendChild(participantOption);
+        }else{
+            participantOption.innerText = `${i} Participants`;
+            particiSelect.appendChild(participantOption);
+        }
```

Comment:
Nice solution for what is often called "pluralization". Interesting fact: This works for English, but will not work for all languages if you ever wanted to translate this website. Some languages have more than two numeric forms (singular, plural). For example, dual, trial, and paucal: https://en.wikipedia.org/wiki/Grammatical_number#Dual.

So when a website undergoes internationalization (the process in which programmers write their code in such a way that the natural language of the UI can be translated by translators) pluralization needs to be extracted to the translation layer rather than the code layer, so that translators of different languages can apply the grammatical rules of their language.